garglk/draw.c

```diff
@@ -212,10 +211,18 @@ static void loadglyph(font_t *f, glui32 cid)
         if (f->make_oblique)
             FT_Outline_Transform(&f->face->glyph->outline, &ftmat);
 
-        if (gli_conf_lcd)
+        if (gli_conf_lcd) {
+            if (gli_conf_gamma == 1.0) {
+                FT_Library_SetLcdFilterWeights(ftlib, filterweights);
```

Comment:
This results in a diagnostic because `FT_LibrarySetLcdFilterWeights()` expects `unsigned char *` but this is passing the equivalent of `const unsigned char*`.  Clearly a bug in the API, but `filterweights` should have the const removed to appease it.

`filterweights` also only used in one place, so it could be moved directly into this block instead of being global, but if you prefer keeping it global, please add `static` to it.

Reply:
I defined it globally because the original equivalent was `#define`d globally. Aand I forgot to remove them.. But they're up at lines 137-139 (originally; now lines 143-145).

GCC doesn't bother me about `const` on my end, but I can definitely change it to be `static` instead of `const` so the tests pass.